build.gradle

```diff
@@ -98,7 +98,6 @@ forbiddenApisTest.enabled = false
 filepermissions.enabled = false
 forbiddenPatterns.enabled = false
 testingConventions.enabled = false
-// Conflicts between runtime kafka-clients:3.0.1 & testRuntime kafka-clients:3.0.1:test
```

Comment:
This comment reference to the line below, can we activate jarhell with 3.0.2?

Reply:
@peternied The kafka change to resolve jar hell was merged into main and didn't make it into the 3.0.2 release.

Here's the change merged into main: https://github.com/apache/kafka/pull/12407

Looks like its still in the code of 3.0.2: https://github.com/apache/kafka/blob/25b1aea02e37da1457026f6ac82a66f81c878e45/build.gradle#L1261

Reply:
Understood, but we should keep the comment until we've resolved the issue it mentions.  If the concern is about the version number being hardcoded to a different version, you could just delete them or replace them with `X.Y.Z`.

Reply:
Yes we should leave the comment in until the jar hell issue is resolved